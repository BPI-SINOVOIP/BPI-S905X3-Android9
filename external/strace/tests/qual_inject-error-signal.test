#!/bin/sh

# Check fault injection along with signal injection.
. "${srcdir=.}/scno_tampering.sh"

run_strace -a12 -echdir,exit_group -einject=chdir:error=ENOENT:signal=USR1 \
	"../$NAME"
match_diff
