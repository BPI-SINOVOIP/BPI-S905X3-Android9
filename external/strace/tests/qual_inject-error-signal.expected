chdir(".")  = -1 ENOENT (No such file or directory) (INJECTED)
--- SIGUSR1 {si_signo=SIGUSR1, si_code=SI_KERNEL} ---
exit_group(0) = ?
+++ exited with 0 +++
